src/main/java/kitchenpos/eatinorders/application/EatInOrderService.java

```diff
@@ -0,0 +1,102 @@
+package kitchenpos.eatinorders.application;
+
+import java.math.BigDecimal;
+import java.util.*;
+import kitchenpos.eatinorders.domain.*;
+import kitchenpos.eatinorders.domain.EatInOrder;
+import kitchenpos.takeoutorders.domain.MenuPriceFoundEvent;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class EatInOrderService {
+  private final EatInOrderRepository orderRepository;
+  private final EatInOrderTableRepository orderTableRepository;
+  private final ApplicationEventPublisher applicationEventPublisher;
+
+  public EatInOrderService(
+      final EatInOrderRepository orderRepository,
+      final EatInOrderTableRepository orderTableRepository,
+      final ApplicationEventPublisher applicationEventPublisher) {
+    this.orderRepository = orderRepository;
+    this.orderTableRepository = orderTableRepository;
+    this.applicationEventPublisher = applicationEventPublisher;
+  }
+
+  @Transactional
+  public EatInOrderResponseDto create(final EatInOrderRequestDto request) {
+
+    final List<EatInOrderLineItemRequestDto> eatInOrderLineItemRequestDtos =
+        request.getEatInOrderLineItemRequestDtos();
+
+    final List<EatInOrderLineItem> eatInOrderLineItems =
+        this.createTakeoutOrderLineItems(eatInOrderLineItemRequestDtos);
+
+    final EatInOrderTable orderTable = findOrderTable(request.getOrderTableId());
+
+    final EatInOrder eatInOrder = EatInOrder.createOrder(eatInOrderLineItems, orderTable);
+    orderRepository.save(eatInOrder);
+    return EatInOrderResponseDto.create(eatInOrder);
+  }
+
+  @Transactional
+  public EatInOrderResponseDto accept(final UUID orderId) {
+
+    final EatInOrder eatInOrder =
+        orderRepository.findById(orderId).orElseThrow(NoSuchElementException::new);
```

Comment:
서비스 메서드마다 반복되는 레포지터리 호출을 공통메서드로 추출하면 가독성을 조금 더 높여 볼 수 있을 것 같아요.
```suggestion
    final EatInOrder eatInOrder = getOrder(orderId)
```
```
    private Order getOrder(final UUID orderId) {
        return orderRepository.findById(orderId)
                .orElseThrow(() -> new NoSuchElementException("주문이 존재하지 않습니다."));
    }
```